packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -350,6 +352,13 @@ function preTransformRequest(server: ViteDevServer, url: string, base: string) {
   // transform all url as non-ssr as html includes client-side assets only
   server.transformRequest(url).catch((e) => {
     // Unexpected error, log the issue but avoid an unhandled exception
```

Comment:
I think this comment should be below the if condition

Reply:
Moved!